src/ucp/core/ucp_request.inl

```diff
@@ -318,6 +318,18 @@ ucp_request_try_send(ucp_request_t *req, unsigned pending_flags)
 static UCS_F_ALWAYS_INLINE void
 ucp_request_send(ucp_request_t *req, unsigned pending_flags)
 {
+#ifndef NVALGRIND
```

Comment:
Why need ifndef?

Reply:
in release mode macro ```VALGRIND_CHECK_MEM_IS_DEFINED``` is defined as ```({(uintptr_t)0;})``` and some compilers reports warning (treated as error) ```statement has no effect```

Reply:
can we fix it somehow in VALGRIND_CHECK_MEM_IS_DEFINED macro?

Reply:
I will try to add inline stub function